tests/k8s/test_config_reload.py

```diff
@@ -0,0 +1,105 @@
+import pytest
+import time
+import k8s_test_utilities as ku
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.platform.processes_utils import wait_critical_processes
+
+
+
+
+def test_config_reload_no_toggle(duthost, k8scluster):
+    """
+    Test case to ensure that when DUT starts as joined to master, and config is saved with disable=false, DUT is still joined to master after config reload
+    
+    Joins master
+
+    Performs config reload
+
+    Ensures that DUT is still joined to master after config reload
+
+    Args:
+        duthost: DUT host object
+        k8scluster: shortcut fixture for getting cluster of Kubernetes master hosts
+    """
+    master_vip = k8scluster.get_master_vip()
+    ku.join_master(duthost, master_vip) # Assertion within to ensure successful join
```

Comment:
Can you make `master_vip` a property of `K8sMasterCluster`? For example:
```
class K8sMasterCluster(object):

    @property
    def vip(self):
        return self.haproxy.mgmt_ip
```
Then the above code can be simpler:
```
ku.join_master(duthost, k8scluster.vip)
```

Reply:
Yes, thank you for the suggestion